src/core_plugins/region_map/public/choropleth_layer.js

```diff
@@ -165,6 +167,7 @@ export default class ChoroplethLayer extends KibanaMapLayer {
 
   }
 
+
```

Comment:
i think you can revert changes to this file, it just adds some newlines, unnecessary noise